src/main/java/uk/nhs/hee/tis/trainee/credentials/filter/SignedDataFilter.java

```diff
@@ -46,20 +49,25 @@ public class SignedDataFilter extends OncePerRequestFilter {
 
   private static final String SIGNATURE_FIELD = "signature";
   private static final String HMAC_FIELD = "hmac";
+  private static final String TIS_ID_FIELD = "tisId";
 
   private final ObjectMapper mapper;
   private final String signatureSecretKey;
 
+  private final RevocationService revocationService;
+
   /**
    * Create a request filter that verifies the signature of signed data.
    *
    * @param mapper             An {@link ObjectMapper} to use when reading the request payload.
    * @param signatureSecretKey The secret key used to sign and verify the signature.
    */
   SignedDataFilter(ObjectMapper mapper,
-      @Value("${application.signature.secret-key}") String signatureSecretKey) {
+      @Value("${application.signature.secret-key}") String signatureSecretKey,
+      RevocationService revocationService) {
```

Comment:
To be pedantic, this should also be listed in the javadoc

Reply:
Oops, missed that 👍🏻